lms/static/js/views/fields.js

```diff
@@ -100,24 +129,24 @@
 
             showNotificationMessage: function(message) {
                 this.$('.u-field-message-help').html('');
-                this.$('.u-field-message-notification').html(message);
+                HtmlUtils.setHtml(this.$('.u-field-message-notification'), message);
             },
 
             showCanEditMessage: function(show) {
                 if (!_.isUndefined(show) && show) {
-                    this.showNotificationMessage(this.getMessage('canEdit'));
+                    this.showNotificationMessage(this.getMessageHTML('canEdit'));
                 } else {
                     this.showNotificationMessage('');
                 }
             },
 
             showInProgressMessage: function () {
-                this.showNotificationMessage(this.getMessage('inProgress'));
+                this.showNotificationMessage(this.getMessageHTML('inProgress'));
             },
 
             showSuccessMessage: function () {
                 var successMessage = this.getMessage('success');
-                this.showNotificationMessage(successMessage);
+                this.showNotificationMessage(this.getMessageHTML('success'));
```

Comment:
Why did this function need to change? As is, the variable `successMessage` is set and then unused.  Maybe you should just change the earlier line to the following, if you just wanted the name change:
```
var successMessage = this.getMessageHtml('success');
```